assets/scss/_partner-form.scss

```diff
@@ -79,7 +79,7 @@
       border: 1px solid #c5cbdb;
       box-sizing: border-box;
       border-radius: 6px;
-      max-width: 350px;
+      width: 350px;
```

Comment:
Why use fixed width?

Reply:
not sure why the input boxes are small all of a sudden, have a better solution in mind?
<img width="614" alt="Screen Shot 2020-12-18 at 8 21 31 AM" src="https://user-images.githubusercontent.com/10093024/102619416-6aaaf800-410a-11eb-8cf4-812e5aaa0b33.png">